doc/classes/SceneTree.xml

```diff
@@ -55,6 +55,7 @@
 				Changes the running scene to a new instance of the given [PackedScene].
 				Returns [constant OK] on success or [constant ERR_CANT_CREATE] if the scene cannot be instantiated.
 				[b]Note:[/b] The scene change is deferred, which means that the new scene node is added on the next idle frame. You won't be able to access it immediately after the [method change_scene_to] call.
+				[b]Note:[/b] Passing a value of [code]null[/code] into the method will unload the current scene without loading a new one. Additionally, passing in any [Object] type that is not a [PackedScene] will be treated as passing [code]null[/code].
```

Comment:
It's good to mention what null does, but I wouldn't add the caveat for passing an invalid Object. That's a GDScript implementation detail and is true for every method in the codebase.

Reply:
That's a fair point! Removed.